src/inference/dev_api/ie_icore.hpp

```diff
@@ -199,6 +199,12 @@ class ICore : public ov::ICore {
     virtual std::map<std::string, std::string> GetSupportedConfig(const std::string& deviceName,
                                                                   const std::map<std::string, std::string>& config) = 0;
 
+    /**
+     * @brief Get configs that are supported by core
+     * @return set of configs that are supported by device
+     */
+    virtual std::set<std::string> QueryCoreSupportedConfig() const = 0;
```

Comment:
Why do we need to extend this API?

Reply:
We need tell plugin the core properties list, then it will be convenient in the future to add new core properties without modification in each plugin.

Reply:
I think it will be better to do with introducing a common property accessor instead of extending our internal API.
Example: https://github.com/openvinotoolkit/openvino/pull/15166